utils/build-script

```diff
@@ -53,6 +53,15 @@ from swift_build_support.swift_build_support.toolchain import host_toolchain
 # TODO: Remove this constant, it's really not helpful.
 HOME = os.environ.get("HOME", "/")
 
+_SUPPORTED_XCODE_BUILDS = (
+    "12A8169g", # 12.0b3
+    "12A8179i", # 12.0b4
+    "12A8189h", # 12.0b5
+    "12A8189n", # 12.0b6
+    "12A7208", # 12.0GM1
+    "12A7209", # 12.0GM2
+    "12A7300", # 12.0.1
+)
```

Comment:
Printing these out directly isn't great. Up for other suggestions, at the very least I could take these comments and use them for the error instead

Reply:
Showing, for example, `12A7300` is not ideal. Switching the comments to values seems ok to me. Another possibility is a single string constant here that contains the version range, like:

```python
_SUPPORTED_XCODE_RANGE = "Xcode 12.0b3 - Xcode 12.0.1"
```

Reply:
I like that idea as well, but I've pushed a version that joins the human readable versions with the build numbers, lmk what you think. The nicer part about this IMO is you only have 1 thing to remember to update.